sw/device/silicon_creator/rom/e2e/ate/BUILD

```diff
@@ -0,0 +1,75 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@//rules:signing.bzl", "offline_presigning_artifacts", "offline_signature_attach")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+load(
+    "//rules/opentitan:defs.bzl",
+    "fpga_params",
+    "opentitan_binary",
+    "opentitan_test",
+)
+
+package(default_visibility = ["//visibility:public"])
+
+opentitan_binary(
+    name = "ate_gpio_toggle_test",
+    testonly = True,
+    srcs = ["ate_gpio_toggle_test.c"],
+    ecdsa_key = {"//sw/device/silicon_creator/rom/keys/fake/ecdsa:prod_key_0_ecdsa_p256": "prod_key_0"},
```

Comment:
Shouldn't we have here the sival key that is already programmed on the ATE OTP?

Reply:
No, this is the fake key so the test can be signed on the fly by opentitantool and run on FPGA for testing. The sival key is used during an offline signing operation and is referenced in the `offline_presigning_artifacts` I added.